library/core/src/panic.rs

```diff
@@ -153,10 +149,7 @@ impl fmt::Display for PanicInfo<'_> {
 
 /// A struct containing information about the location of a panic.
 ///
-/// This structure is created by the [`location`] method of [`PanicInfo`].
-///
-/// [`location`]: ../../std/panic/struct.PanicInfo.html#method.location
-/// [`PanicInfo`]: ../../std/panic/struct.PanicInfo.html
+/// This structure is created by the [`PanicInfo::location`] method of [`PanicInfo`].
```

Comment:
nit: this would be better worded as
```suggestion
/// This structure is created by [`PanicInfo::location()`].
```